FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h

```diff
@@ -171,7 +171,10 @@ NS_SWIFT_NAME(RemoteConfigSettings)
 @end
 
 #pragma mark - FIRRemoteConfigUpdate
-/// Firebase Remote Config update
+/// Used by Remote Config real-time config update service, this class represents changes between the
+/// newly fetched config and the current one. An instance of this class is passed to
+/// `FIRRemoteConfigUpdateCompletion` whenever there has been a successful auto-fetch and there is
```

Comment:
```suggestion
/// `FIRRemoteConfigUpdateCompletion` when a new config version has been automatically fetched
```

Reply:
I think auto-fetch might be too internal to the SDKs to use in the docs

Reply:
This is a great descriptive comment, btw!

Reply:
Updated